src/database/migrations/20210625081817-create-student.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable('Students', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      firstname: {
+        type: Sequelize.STRING,
+        allowNull: false
+      },
+      surname: {
```

Comment:
@xx3u лучше назвать поля в одном стиле firstName, lastName, middleName
```suggestion
      lastName: {
```

Reply:
Исправлено 
![image](https://user-images.githubusercontent.com/45433350/123503591-bb7d4980-d675-11eb-8e45-a109c3e36536.png)